SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -735,7 +735,11 @@ bool RunSchemaCompiler()
 	FString PluginDir = FSpatialGDKServicesModule::GetSpatialGDKPluginDirectory();
 
 	// Get the schema_compiler path and arguments
-	FString SchemaCompilerExe = FPaths::Combine(PluginDir, TEXT("SpatialGDK/Binaries/ThirdParty/Improbable/Programs/schema_compiler.exe"));
+#if PLATFORM_WINDOWS
```

Comment:
Nit: Maybe moving the definition of `SchemaCompilerExe` to the beginning of the file near `RelativeSchemaDatabaseFilePath` would make it look nicer? Also might be nice to put these constants in anonymous namespace.